cmd/simulator/main.go

```diff
@@ -41,6 +41,7 @@ import (
 )
 
 var (
+	pdAddr         = flag.String("pd", "", "pd address")
```

Comment:
Better to use a default value 'http://127.0.0.1:2379'

Reply:
The thing is that if we use the default value here, we cannot run all cases when `make simulator` due to the server is not clean. Since this option is mainly for some specific scenarios, I think leave it as an empty string is ok. What do you think?

Reply:
oh, you are right. My mistake.